site/content/docs/5.2/content/tables.md

```diff
@@ -64,20 +77,47 @@ Use contextual class to color tables, table rows or individual cells.
 
 ## Accented tables
 
+### Striped rows
+
+Use `.table-striped` to add zebra-striping to any table row within the `<tbody>`.
+
+{{< table class="table table-striped" caption="Boosted striped rows table" >}}
+
+### Striped columns
+
+Use .table-striped-columns to add zebra-striping to any table column.
+
+{{< table class="table table-striped-columns" caption="Boosted striped columns table" >}}
+
+These classes can also be added to table variants:
+
+{{< table class="table table-dark table-striped" caption="Boosted dark striped rows table" >}}
+
+{{< table class="table table-dark table-striped-columns" caption="Boosted dark striped columns table" >}}
+
+<!-- Boosted mod : no .table-success -->
+
 ### Hoverable rows
 
 Add `.table-hover` to enable a hover state on table rows within a `<tbody>`.
 
-{{< table class="table table-hover" >}}
+{{< table class="table table-hover" caption="Boosted hoverable table" >}}
+
+{{< table class="table table-dark table-hover" caption="Boosted hoverable dark table" >}}
+
+These hoverable rows can also be combined with the striped rows variant:
+
+{{< table class="table table-striped table-hover" caption="Boosted hoverable striped table" >}}
 
-{{< table class="table table-dark table-hover" >}}
+{{< table class="table table-dark table-striped table-hover" caption="Boosted hoverable dark striped table" >}}
 
 ### Active tables
 
 Highlight a table row or cell by adding a `.table-active` class.
 
 <div class="bd-example">
   <table class="table">
+    <caption class="visually-hidden">Boosted table with an active row and cell</caption>
```

Comment:
In other Markdown documents captions are not added:
- https://deploy-preview-859--boosted.netlify.app/docs/5.2/components/back-to-top/#sass-options
- https://deploy-preview-859--boosted.netlify.app/docs/5.2/content/typography/#headings
- https://deploy-preview-859--boosted.netlify.app/docs/5.2/content/typography/#regular-text
- https://deploy-preview-859--boosted.netlify.app/docs/5.2/forms/quantity-selector/#events
- https://deploy-preview-859--boosted.netlify.app/docs/5.2/layout/grid/#grid-options
- https://deploy-preview-859--boosted.netlify.app/docs/5.2/utilities/vertical-align/
- All `{{< bs-table >}}`

Just asking, do we need to add captions? Mandatory in this case?

Reply:
I think it would be better, just wondering if we should add this in this PR or not 🤔 I don't want those to be blocking, so tell me :)

Reply:
OK so let's tackle it in another PR and check with a11y experts if it's needed. Please create an issue and ask them :)